tools/mdm/apple/toggle-mdm-dev.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# To toggle MDM, run `source toggle-mdm-dev`
+
+if [[ $USE_MDM == "1" ]]; then
+export USE_MDM=0
+else
+export USE_MDM=1
+fi
+
+source $FLEET_ENV_PATH
```

Comment:
Cool scripts!!

Reply:
oops, didn't mean to include those here!